MIP0/mip0.md

```diff
@@ -7,287 +7,252 @@ Title: The Maker Improvement Proposal Framework
-	-   The MIP Editor(s) controls phase 2 of the MIP lifecycle and can assign MIP numbers
-	-   The MIP Editor(s) is an admin on the MIPs Github repository
-	-   The MIP Editor(s) is a moderator on the MIPs Discourse forum
-	-   The MIP Editor(s) is responsible for updating the status of MIPs, as described in MIP0c4 “The MIP Lifecycle”.
--   **Governance Facilitator:** Operates voting frontends, runs governance meetings and accepts MIPs that are ready to be included in the Governance Cycle and thus, voted on.
--   **Specific authority of the Governance Facilitator in MIP0 processes:**
+#### MIP Editor
+MIP Editors enforce the administrative and editorial aspects of the overall MIPs process and program.
 
-	-   Consensus from all governance facilitators controls phase 6 of the MIP lifecycle, which allows them to, with consensus, add valid MIPs to the inclusion poll of the next governance cycle, moving them from phase 5 (formal submission) to phase 7 (governance cycle).
+##### Specific authority of the MIP Editor(s) in MIP0 processes
 
-Personnel may be added to these roles using a MIP0c10 sub-proposal.
-Personnel may be removed from these roles using a MIP0c11 sub-proposal.
+* MIP Editors control phase 2 of the MIP lifecycle and can assign MIP numbers.
+* MIP Editors are admins on the [MIPs Github repository](https://github.com/makerdao/mips).
+* MIP Editors moderate the [MIPs category in the forum](https://forum.makerdao.com/c/mips/14).
+* MIP Editors are responsible for updating the status of MIPs, as described in `MIP0c4`, *The MIP Lifecycle*.
 
----
-### MIP0c9: Core Personnel Role List 
+#### Governance Facilitator
+Governance Facilitators operate voting frontends, run governance meetings and accept MIPs that are ready to be included in the Governance Cycle and thus voted on.
 
-This list can be amended through the core personnel onboarding (MIP0c12) and offboarding components (MIP0c13) of MIP0.
+Governance Facilitators have all the editorial powers a MIP Editor has, and can bestow these powers on contributors.
 
-Entries into this list should follow the following template:
+##### Specific authority of the Governance Facilitator in MIP0 processes
 
-```
-- Person Name: The name of the person in the core role.
-	- Sub-proposal Number (MIP0c12-SP): #
-	- Core Role: The core role in which the person operates.
-	- Date Added: <date in (yyyy-mm-dd) format>
-```
+* Consensus from all Governance Facilitators controls phase 6 of the MIP lifecycle, which allows them to add valid MIPs to the inclusion poll of the next Governance Cycle, moving them from phase 5 (Formal Submission) to phase 7 (Governance Cycle).
 
-**Active Core Personnel List:**
+Governance Facilitators are defined as Facilitators attached to Core Units with the ID `GOV` in [MIP38: DAO Primitives](https://mips.makerdao.com/mips/details/602afa75fa09da0714da2569). They are added using [MIP41: Facilitator Framework](https://mips.makerdao.com/mips/details/602afa75fa09da0714da256c).
```

Comment:
We should come up with a policy for links here. I suspect that we should link directly to github, and then have the portal replace these links magically.